doc/rules-format.md

```diff
@@ -58,12 +59,14 @@ Kccgst       ::= "keycodes" | "symbols" | "types" | "compat" | "geometry"
 Rule         ::= { MlvoValue } "=" { KccgstValue } "\n"
 MlvoValue    ::= "*" | GroupName | <ident>
 KccgstValue  ::= <ident>
+```
 
 Notes:
 
 - Include processes the rules in the file path specified in the ident,
-  in order. %-expanasion is performed, as follows:
+  in order. %-expansion is performed, as follows:
 
+```
```

Comment:
This would maybe be better as a (sub)list? (With just the `%H` parts in ticks)

Reply:
I had that first but it doesn't work, the % gets swallowed if the backticks and the `%H` is on the same line. This is a doxygen markdown issue, not visible in gitlab markdown. This only applies to letters, not to `%%` or `%-` in the sentence above. Backslash escape works (`\%H`) but that is confusing in the .md source then.

I'd rather go with bad rendering than ambiguity here, at least the former doesn't create bug reports :)